lib/ClangImporter/ImportDecl.cpp

```diff
@@ -7903,9 +7903,39 @@ createUnavailableDecl(Identifier name, DeclContext *dc, Type type,
   return var;
 }
 
+static ExtensionDecl *figureOutTheDeclarationContextToImportInto(Decl* D, DeclContext *&DC, IterableDeclContext *&IDC) {
+  if (auto nominal = dyn_cast<NominalTypeDecl>(D)) {
+    DC = nominal;
+    IDC = nominal;
+    return nullptr;
+  }
+  else {
+    ExtensionDecl *ext = cast<ExtensionDecl>(D);
```

Comment:
Can you use auto * here. The general LLVM style is if the type is already on the same line (i.e. when you are using cast/dyn_cast) you should use auto *, but beyond that and iterators, LLVM style says avoid auto.

https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable